src/main/java/edu/psu/swe/commons/jaxrs/utilities/JsonSerializationUtil.java

```diff
@@ -37,16 +38,16 @@
     return Optional.of(obj);
   }
   
-  public static <T> Optional<List<T>> deserializeList(String json, Class<T> clazz) throws IOException {
+  public static <T> List<T> deserializeList(String json, Class<T> clazz) throws IOException {
     if (json == null || json.trim().isEmpty()) {
-      return Optional.empty();
```

Comment:
Are there projects using this method at the moment?  We'll need to know what (if anything) this will break.

Reply:
I am only aware of Worklion using this class. I will clean it up during the next release.